### ISSUE TITLE
Display of default ACLs at the root folder level after creation of the workspace

### ISSUE THREAD

**Issue by Jerome-Herbinet** [arawa/workspace#509]
When you create a workspace and go into Nextcloud Files to deploy the tree structure that the root folder will contain, you are currently obliged to add the basic ACLs of the "users" and "space manager" groups (to the root folder) in order to be able to find and refine them in all the sub-folders of the workspace (which is not intuitive and can create confusion and oversights) ... but also, for example, to slightly restrict the default rights of the "user group" by removing the right to share.
In other words, it would be nice, when you arrive in the root folder of the workspace (immediately after the creation of the workspace by the general manager) that the 2 ACLs are already displayed, with all rights open (Read, Write, Create, Delete, Share).

@LivOriona @zak39

**Comment by dorianne-arawa** [comment]
I think we already talked about this. I'm not sure it's possible for the moment. 
Does anything changed since this time @zak39?
